examples/ucp_hello_world.c

```diff
@@ -633,6 +631,9 @@ static void print_usage()
             "ucp_worker_get_efd function with later poll\n");
     fprintf(stderr, "  -b      Select test mode \"busy polling\" to test "
             "ucp_tag_probe_nb and ucp_worker_progress (default)\n");
+    fprintf(stderr, "  -n name Set node name or IP address "
+            "of the server (required for client and should be ignored "
+            "for server)\n");
```

Comment:
why need to move it away from common_help? since it's added to both UCT and UCP hello world?

Reply:
It's not common anymore for all examples, since I've reused this code for client-server as well